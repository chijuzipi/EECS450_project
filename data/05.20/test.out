Generating the available host list.....
