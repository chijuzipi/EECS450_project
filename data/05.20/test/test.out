Generating the available host list.....
error!! No = in cookies
 cookiesavailable
error!! No = in cookies
 cookiesavailable
67579
43507
host number 955
token number 186419
Finish loading data from httpfox.sqlite
43074
26338
host number 616
token number 101249
Finish loading data from httpfox.sqlite2
The original sequence number for xvideos.com is 1085
Cleaning the overlap values of the two databases....
The new sequence number for xvideos.com is 773
====================Finding Sub-string Start====================
Host 1 / 1: xvideos.com
The number of strings:773
The terminator is :  Ï¨
Terminator is determined
Start buidling the tree....
----------------------------------------------------------------------
